pipeline_dp/combiners.py

```diff
@@ -64,6 +64,10 @@ def compute_metrics(self, accumulator):
     def metrics_names(self) -> List[str]:
         """Return the list of names of the metrics this combiner computes"""
 
+    @abc.abstractmethod
+    def explain_computation(self) -> str:
```

Comment:
IIUC, the type annotation should be something like Union[Callable, str] or Union[Callable, str, Sequence[Callable]]

Reply:
yeah, thanks